README.md

```diff
@@ -60,3 +60,20 @@ For accessibilit testing:
 ```bash
 $ yarn test:a11y
 ```
+For code coverage:
+```bash
+$ yarn test:coverage
+$ yarn sonar-scanner
```

Comment:
I don't think people should be running this locally?
yarn test:coverage should be enough?

Reply:
Also missing changes adding this into Jenkins

Reply:
We have two separate tickets for addressing above problems
ROC-2325
ROC-2326

Reply:
regardless `yarn sonar-scanner` shouldn't be used by developers so it shouldn't be here

Reply:
as part of 2326 we can add this to stage to jenkins

Reply:
you're adding a line to the README that shouldn't be there, that's not how developers should be seeing code coverage.

`$ yarn test:coverage` is

Reply:
can you suggest me how you want me to change README?